.github/workflows/PR-build.yml

```diff
@@ -134,6 +134,9 @@ jobs:
     runs-on: ubuntu-latest
     needs: [changes]
     steps:
+    - name: Remove cache
+      run: rm -rf /opt/hostedtoolcache
```

Comment:
Add this to CI build job also, just in case.

Reply:
Added to CI in my latest commit